README.md

```diff
@@ -1,7 +1,6 @@
 # ID Obfuscator (beta)
 
-[![Build Status](https://travis-ci.org/craftwork/id-obfuscator.svg?branch=master)](https://travis-ci.org/craftwork/id-obfuscator)
-[![codecov](https://codecov.io/gh/craftwork/id-obfuscator/branch/master/graph/badge.svg)](https://codecov.io/gh/craftwork/id-obfuscator)
+[![Build Status](https://travis-ci.org/craftwork/id-obfuscator.svg?branch=master)](https://travis-ci.org/craftwork/id-obfuscator) [![codecov](https://codecov.io/gh/craftwork/id-obfuscator/branch/master/graph/badge.svg)](https://codecov.io/gh/craftwork/id-obfuscator) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/craftwork/id-obfuscator/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/craftwork/id-obfuscator/?branch=master)
```

Comment:
I'd rather new-line each badge, just so it's easily readable/modifiable in an IDE